EstateAdministrationUI.IntegrationTests/Common/TestingContext.cs

```diff
@@ -0,0 +1,87 @@
+﻿namespace EstateAdministrationUI.IntegrationTests.Common
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using SecurityService.DataTransferObjects.Responses;
+    using Shared.IntegrationTesting;
+    using Shared.Logger;
+    using Shouldly;
+    using TechTalk.SpecFlow;
+
+    public class TestingContext
+    {
+        public EstateDetails GetEstateDetails(TableRow tableRow, Guid? testId = null)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use the overloading mechanism instead of the optional parameters.](https://app.codacy.com/manual/stuart_ferguson1/EstateAdministrationUI/pullRequest?prid=5485852)